src/app_service/service/message/service.nim

```diff
@@ -642,7 +644,21 @@ QtObject:
       for id in messagesIdsArr:
         messagesIds.add(id.getStr)
 
-    let data = MessagesMarkedAsReadArgs(chatId: chatId, allMessagesMarked: false, messagesIds: messagesIds)
+    var count: int
+    discard responseObj.getProp("count", count)
+
+    if count != len(messagesIds):
```

Comment:
is it may be not 100% true condition :) if `count > len(messagesIds)` that means all messages has been marked :)